tasks.py

```diff
@@ -147,7 +147,7 @@ def docker_compose(context, command, **kwargs):
         **kwargs: Passed through to the context.run() call.
     """
     compose_command_tokens = [
-        "docker-compose",
+        "docker compose",
```

Comment:
nit: there's at least three other instances of `docker-compose` in this file, though in comments/docstrings/logs rather than execution path. Might as well change them too for consistency?